cmd/gapit/flags.go

```diff
@@ -141,8 +141,6 @@ type (
 		CaptureID bool `help:"if true then interpret the capture file argument as a capture ID that is already loaded in gapis"`
 	}
 	CommandFilterFlags struct {
```

Comment:
Shall we just get rid of the whole struct?

Reply:
ditto